src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1162,7 +1162,7 @@ def update_app_service_plan(instance, sku=None, number_of_workers=None,
 
     instance.sku = sku_def
     if admin_site_name is not None:
-        Logger.warning("The 'admin_site_name' property has been deprecated")
+        logger.warning("The 'admin_site_name' property has been deprecated")
```

Comment:
This should use the CLI's built-in deprecation mechanisms.

Reply:
@tjprescott can you point me to an example of this when deprecating a property - the examples i saw were all where the command was being deprecated. thank you!

Reply:
@panchagnula, please see https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments

Reply:
thank you.